x/amm/spec/05_slippage.md

```diff
@@ -0,0 +1,7 @@
+# Slippage model
+
+On oracle pool, to prevent bots from splitting the trades into smaller pieces to reduce slippage applied on the trade, we introduce the stacked slippage model where slippage is stacked for a block.
+
+Oracle pool has a dynamic weight model, the weight is updated before starting the swap transactions execution and does not change until the end of the block.
```

Comment:
Oracle based pools use balancer slippage model to prevent arbitrage opportunity on swaps with other DEXs.
It is calculating slippage from balancer model - weight and liquidity on the pool.

When the weight is changed based on oracle price, pool weights are updated based on oracle price, and for that, price on the pool is exactly same as oracle price.